test/Sentry.Tests/SentryLogTests.cs

```diff
@@ -0,0 +1,260 @@
+using System.Text.Encodings.Web;
+
+namespace Sentry.Tests;
+
+/// <summary>
+/// <see href="https://develop.sentry.dev/sdk/telemetry/logs/"/>
+/// </summary>
+public class SentryLogTests
+{
+    private static readonly DateTimeOffset Timestamp = new(2025, 04, 22, 14, 51, 00, TimeSpan.FromHours(2));
+    private static readonly SentryId TraceId = SentryId.Create();
+    private static readonly SpanId? ParentSpanId = SpanId.Create();
+
+    private static readonly ISystemClock Clock = new MockClock(Timestamp);
+
+    private readonly IDiagnosticLogger _output;
+
+    public SentryLogTests(ITestOutputHelper output)
+    {
+        _output = new TestOutputDiagnosticLogger(output);
+    }
+
+    [Fact]
+    public void WriteTo_Envelope_MinimalSerializedSentryLog()
+    {
+        var options = new SentryOptions
+        {
+            Environment = "my-environment",
+            Release = "my-release",
+        };
+
+        var log = new SentryLog(Timestamp, TraceId, SentryLogLevel.Trace, "message");
+        log.SetAttributes(options);
+
+        var envelope = Envelope.FromLog(log);
+
+        using var stream = new MemoryStream();
+        envelope.Serialize(stream, _output, Clock);
+        stream.Seek(0, SeekOrigin.Begin);
+        using var reader = new StreamReader(stream);
+        var header = JsonDocument.Parse(reader.ReadLine()!);
+        var item = JsonDocument.Parse(reader.ReadLine()!);
+        var payload = JsonDocument.Parse(reader.ReadLine()!);
+
+        reader.EndOfStream.Should().BeTrue();
+
+        header.ToIndentedJsonString().Should().Be($$"""
```

Comment:
This is the sort of thing Verify is good at testing... might be a good excuse to get familiar with that.

Reply:
More "default-attributes" have been added to the Logs-specification, which I will add in a follow-up PR.
During that, will refactor to use Verify here as well.